jaxrs-api/src/main/java/se/fortnox/reactivewizard/jaxrs/WebException.java

```diff
@@ -155,7 +155,7 @@ private Level logLevelFromStatus(HttpResponseStatus httpStatus) {
         if (httpStatus.code() >= 500) {
             return Level.ERROR;
 
-        } else if (httpStatus == HttpResponseStatus.NOT_FOUND) {
+        } else if (httpStatus.equals(HttpResponseStatus.NOT_FOUND)) {
```

Comment:
nice since httpstatus is not an enum. :+1:

Reply:
Have you done some search if we do the same error somewhere else?

Reply:
🤯 Nice find!

Reply:
> Have you done some search if we do the same error somewhere else?

Just checked, nothing found in this repo anyway 👍